server/ranking/ranking.go

```diff
@@ -0,0 +1,92 @@
+package ranking
+
+import (
+	"encoding/hex"
+	"errors"
+	"strconv"
+	"time"
+
+	"github.com/golang/protobuf/ptypes/empty"
+	"github.com/iotexproject/iotex-election/committee"
+	pb "github.com/iotexproject/iotex-election/pb/ranking"
```

Comment:
github.com/iotexproject/... shoud be grouped together,
other github or golang imports in another group